crates/node/core/src/args/network.rs

```diff
@@ -130,7 +130,16 @@ impl NetworkArgs {
         secret_key: SecretKey,
         default_peers_file: PathBuf,
     ) -> NetworkConfigBuilder {
-        let chain_bootnodes = chain_spec.bootnodes().unwrap_or_else(mainnet_nodes);
+        let chain_bootnodes = self
```

Comment:
```suggestion
        let mut chain_bootnodes = chain_spec.bootnodes().unwrap_or_default();
```
always add the default bootnodes, otherwise we need a new cli arg flag `--no-default-bootnodes`

Reply:
hmm, this change would require --no-default-bootnodes
imo --bootnodes should override default bootnodes

Reply:
update @emhane